### DISCUSSION TITLE
2005/05/16/%E5%A5%BD%E7%B4%AF%E5%95%8A%EF%BC%81

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#29] [Announcements]
# 2005/05/16/%E5%A5%BD%E7%B4%AF%E5%95%8A%EF%BC%81

我还得再休息两天

https://blog.qizhen.xyz/2005/05/16/%E5%A5%BD%E7%B4%AF%E5%95%8A%EF%BC%81

**Comment by ruanqizhen** [comment]
Qizhen
2005年05月19日 06:56 编辑
半鱼生涯的Jessica，你的页面进不去啊。

回复
Qizhen
2005年05月19日 06:55 编辑
我的工作是软件开发一类的 :)

回复
小＆U
2005年05月18日 21:19 编辑
多休息休息，然后就可以好好的玩了~

回复
妤
2005年05月17日 19:55 编辑
想问你是做什么的哦。。。。。

回复
Jessica
2005年05月17日 11:37 编辑
我座了板凳。我试着添加音乐了。却在最后一步失败了。不知道为什么。奇怪。。。你什么时候能休息完？知道偶一下。。。谢谢先。

回复
candy
2005年05月17日 09:39 编辑
哈哈，沙发沙发先！然后再看看最近小四眼在忙些什么！嘻嘻

回复
